schema/crdb/auto-restart-policy-v2/README.md

```diff
@@ -0,0 +1,41 @@
+# Overview
```

Comment:
I think it's possible to create tests that exercise migrations like this that have data in them.  What do you think of adding one?  (I haven't actually done this -- I just recently learned there's some infrastructure for this.)

Reply:
I'm happy to do that, do you know where I  ought to look to find said infrastructure?

Reply:
Have a look hereabouts: https://github.com/oxidecomputer/omicron/blob/40fc38351e8ad6a639968376de7fcc0ccee1bef4/nexus/tests/integration_tests/schema.rs#L1369

Reply:
lovely, thank you!

Reply:
Added data migration tests for this (and the earlier `boot_on_fault` -> `instance_auto_restart` migration) in https://github.com/oxidecomputer/omicron/pull/6503/commits/dc11190e70a17aac9522a2c38caa9505dcb012cb.